src/styles/main.css

```diff
@@ -0,0 +1,303 @@
+}
+
+.main .apartment .sec-head {
+  font-size: 36px;
+  line-height: 42px;
+}
+
+.main .apartment .sub-head {
+  font-size: 16px;
+  line-height: 19px;
+  color: #687c94;
+}
+
+.main .apartment .host-button {
+  width: 150px;
+  height: 50px;
+  outline: none;
+  font-size: 16px;
+  color: #118488;
+  background-color: white;
+  border: 1px solid #118488;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  border-radius: 4px;
+}
+
+.main .apartment .apart-details {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.main .apartment .apart-details p {
+  padding-right: 82px;
+  font-size: 16px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.main .apartment .apart-info {
+  font-size: 16px;
+  line-height: 24px;
+  color: #3d4e61;
+}
+
+.main .apartment .contact-host {
+  font-weight: bolder;
+}
+
+.main .apartment .amenity-head {
+  font-weight: bolder;
+}
+
+.main .apartment .facilities span {
+  margin-left: 120px;
+}
+
+.main .apartment .show-all {
+  font-weight: bolder;
+}
+
+.main form {
+  height: 330px;
+  width: 319px;
+  padding: 30px 30px 38px;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  border: 1px solid #E1E7ED;
+  border-radius: 3px;
+}
+
+.main form .price {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3D4E61;
+}
+
+.main form input {
+  width: 257px;
+  height: 50px;
+  outline: none;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  font-size: 16px;
+  line-height: 22px;
+  border-radius: 3px;
+  border: 1px solid #E1E7ED;
+  padding: 14px 17px;
+  margin-bottom: 16px;
+}
+
+.main .form-button {
+  width: 257px;
+  height: 50px;
+  outline: none;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  font-size: 16px;
+  line-height: 22px;
+  border-radius: 4px;
+  background: #FD5C63;
+  border: 1px solid #FD5C63;
+  color: white;
+  margin-top: 8px;
+}
+
+.main .review .rev-block {
+  position: relative;
+}
+
+.main .review .rev-block::before {
+  content: "";
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+  background-image: url(../images/headshot.jpg);
+  position: absolute;
+}
+
+.main .review .rev-block .name {
+  padding-left: 80px;
+  font-weight: bolder;
+}
+
+.main .review .rev-block .date {
+  padding-left: 80px;
+}
+
+.foot {
+  background: #FFFFFF;
+  -webkit-box-shadow: 0px -1px 0px #E1E7ED;
+          box-shadow: 0px -1px 0px #E1E7ED;
+  border: 1px solid  #E1E7ED;
+  padding: 40px calc(50% - 451px) 25px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.foot .foot-nav {
+  list-style: none;
+  padding: 0;
+  margin-right: 300px;
+}
+
+.foot .foot-nav .bnb-items {
+  margin-bottom: 15px;
+}
+
+.foot .foot-nav .bnb-items a {
+  text-decoration: none;
+  font-size: 16px;
+  color: #3D4E61;
+}
+
+.foot .foot-nav .bnb-items a.link-head {
+  display: block;
+  font-weight: bolder;
+  margin-bottom: 30px;
+}
+
+@media screen and (max-width: 950px) {
```

Comment:
it looks very bad

![image](https://user-images.githubusercontent.com/44294945/67405896-09e36680-f5be-11e9-91ed-86fa4ba8984b.png)

Reply:
change it, look how other people do this task